### DISCUSSION TITLE
Are these Many tiny black holes are normal?

### DISCUSSION THREAD

**Discussion by Liriel-666** [pmgducati/Blue-Retro-AIO-Units#71] [General]
Are these Many tiny black holes are normal?
for what are these?

**Comment by pmgducati** [comment]
Can you post a picture of what you are referring to?

Thanks

**Comment by Liriel-666** [comment]
![capture_001_24112023_200214](https://github.com/pmgducati/Blue-Retro-AIO-Units/assets/10562572/26689777-66e1-47d1-b100-b26f13b0e7fb)
when i load the pcbs in the print service like oshpark or jlcpcb these many black dots show as drill holes.
On your Picture of the PCBs on the Github Main page there are these black spots on the pcb too

**Reply by pmgducati** [reply]
They are stitching via's, the connect the ground planes together on both sides.  Depending on the PCB manufacturer they may or may not be visible.  But is it normal.
